src/lib.rs

```diff
@@ -105,7 +95,38 @@ pub fn get_zones(pbf: &mut OsmPbfReader) {
                 tags: vec![],
             };
 
-            println!("{:?}", zone);
+            // Ignore zone without boundary polygon
+            if zone.boundary.is_none() {
+                continue;
+            }
+
+            zone.admin_level.map(|level| {
+                let count = counts.entry(level).or_insert(0);
+                *count += 1;
+            });
+            zones.push(zone);
         }
     }
+
+    let stats = CosmogonyStats {
```

Comment:
wouldn't it be more convenient to create those stats above and mutate them ? it will be easier when we'll add more stats

Reply:
Done